variable-explorer.md

```diff
@@ -97,6 +116,12 @@ local variables defined at the current execution point: `country_data` and
 
 ![](media/RTVS-REPL-variable-explorer-view-locals.png)
 
+We support viewing variables defined inside of private named scopes as well
+(i.e., variables defined within a package). For example, here are the variables
+defined in the Shiny package:
+
+![](media/04_variable_explorer_scopes.png)
```

Comment:
In fact, is it even needed? If you look at the beginning of "Scopes", it already talks about how you can view variables inside packages - this just repeats that.